docworks-cli/test/config-file.unit.spec.js

```diff
@@ -0,0 +1,251 @@
+const chai = require('chai')
```

Comment:
we should open a tech-debt task to move docworks to jest. Could be a candidate first-task for a newcomer